integrations/database/mariadb/README.md

```diff
@@ -0,0 +1,18 @@
+# Generating the KubeArchive schema file.
```

Comment:
As this is a title, it needs title capitalization:

```suggestion
# Generating the KubeArchive Schema File.
```

Reply:
Fixed.